tests/components/russound_rio/test_config_flow.py

```diff
@@ -117,3 +118,56 @@ async def test_import_cannot_connect(
 
     assert result["type"] is FlowResultType.ABORT
     assert result["reason"] == "cannot_connect"
+
+
+async def _start_reconfigure_flow(
+    hass: HomeAssistant, mock_config_entry: MockConfigEntry
+) -> ConfigFlowResult:
+    """Initialize a reconfigure flow."""
+    mock_config_entry.add_to_hass(hass)
+
+    reconfigure_result = await mock_config_entry.start_reconfigure_flow(hass)
+
+    assert reconfigure_result["type"] is FlowResultType.FORM
+    assert reconfigure_result["step_id"] == "reconfigure"
+
+    return await hass.config_entries.flow.async_configure(
+        reconfigure_result["flow_id"],
+        MOCK_RECONFIGURATION_CONFIG,
+    )
+
+
+async def test_reconfigure_flow(
+    hass: HomeAssistant,
+    mock_russound_client: AsyncMock,
+    mock_setup_entry: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test reconfigure flow."""
+
+    result = await _start_reconfigure_flow(hass, mock_config_entry)
+
+    assert result["type"] is FlowResultType.ABORT
+    assert result["reason"] == "reconfigure_successful"
+
+    entry = hass.config_entries.async_get_entry(mock_config_entry.entry_id)
+    assert entry
+    assert entry.data == {
+        CONF_HOST: "192.168.20.70",
+        CONF_PORT: 9622,
+    }
```

Comment:
```suggestion
    assert entry.data == MOCK_RECONFIGURATION_CONFIG
```

Reply:
I think I disagree with this comment.
I think asserts are clearer if the data doesn't use complex constants.

Reply:
if you are fine with it @epenet, I won't argue. I like consts here because you then only need to change it in one place.